contract/src/test/java/io/neow3j/contract/SmartContractTest.java

```diff
@@ -244,21 +250,27 @@ public void callFunctionWithParams() throws IOException {
                 is(BigInteger.valueOf(3000000000000000L)));
     }
 
+    @Ignore("The test fails because the mocked `incokescript` RPC call expects the script hash of "
+            + "the sender in as a parameter in the list of signers. But because of changed "
+            + "InteropServiceCalls, neow3j currently produces another script hash from the sender"
+            + " account as expected in this test. As soon as these changes to the "
+            + "InteropServiceCode are introduced to neow3j this test can be normally run.")
```

Comment:
We just need to remember to remove the ignored of this test when @mialbu completes the adaptions.
Should we add a TODO here? No worries if not.